public_src/core/editingOptions.interface.ts

```diff
@@ -10,3 +10,12 @@ export interface ITagBrowserOptions {
   limitedKeys: boolean;
   makeKeysReadOnly: boolean;
 }
+
+export interface ISuggestionsBrowserOptions {
```

Comment:
maybe we should later rename Suggestions to advice/hint/tip or something shorter like those... Also, we should probably rewrite different keywords to shorter and obvious equivalents - options -> opts; error -> err, event -> evt,...